CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+cmake_minimum_required(VERSION 3.9)
```

Comment:
Most probably the required cmake version can be lowered a lot, since this `CMakeLists.txt` is very simple, and it does not use many advanced features. IMHO 3.0 should be already good enough, as I do not remember which version introduced the `LINK_PUBLIC` feature for `target_link_libraries`.